gocop/storer/sql.go

```diff
@@ -0,0 +1,167 @@
+package storer
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	_ "github.com/lib/pq" // import postgres driver
+)
+
+type SQL struct {
+	db *sql.DB
+}
+
+// NewSQL creates a new SQL storer instance. Currently hardcoded for postgres databases.
+func NewSQL(host, port, user, password, dbname, sslmode string) (Storer, error) {
```

Comment:
It is probably worth calling this a NewPSQL to qualify what SQL type this factory is making for the storer abstraction